src/content/docs/en/guides/client-side-scripts.mdx

```diff
@@ -73,18 +84,18 @@ To avoid bundling the script, you can use the `is:inline` directive.
 ```
 
 :::note
-Astro's default bundling behavior will be disabled in some situations. In particular, adding `type="module"` or any attribute other than `src` to a `<script>` tag will cause Astro to treat the tag as if it had an `is:inline` directive. The same will be true when the script is written in a JSX expression.
+Astro will not process your script tags in some situations. In particular, adding `type="module"` or any attribute other than `src` to a `<script>` tag will cause Astro to treat the tag as if it had an `is:inline` directive. The same will be true when the script is written in a JSX expression.
 :::
 
 📚 See our [directives reference](/en/reference/directives-reference/#script--style-directives) page for more information about the directives available on `<script>` tags.
 
-### Script loading
+### Include javascript files on your page
```

Comment:
```suggestion
### Running scripts from JavaScript files
```
I'm still not entirely sure about the wording, but something more like this seems to most precisely describe the action? Do you have something better for here?

Reply:
I was thinking of the case where users have a javascript file written somewhere (`src/lib/smooth-scroll.js`). I think they would lookup how to "include".

Reply:
OK, happy to defer to you on this one!

Reply:
The examples below show discrete files as well, so I feel good about "include".